Wbooks/Main/MainView.swift

```diff
@@ -11,25 +11,18 @@ import UIKit
 final class MainView: NibView {
     
     // MARK: Properties
-    @IBOutlet weak var userLabel: UILabel! {
+    var mainViewModel: MainViewModel! {
```

Comment:
Let's move this property to `MainViewController`. The `didSet` part can be transformed into a method `setup(with viewModel: MainViewModel)` and then be called from `MainViewController.viewDidLoad()`.

Reply:
`setup(with viewModel: MainViewModel)` would be located in `MainView`